src/styles/app.scss

```diff
@@ -324,3 +324,58 @@ header h1 {
   }
 }
 
+@media (prefers-color-scheme: dark) {
+  $yellow: #835f00;
+  $body-bg: #181a1b;
+  $body-fg: #e8e6e3;
+  $font-color: #d5cbc6;
+
+  body {
```

Comment:
Hum... You are using SCSS variables when you should use HTML variables. With the current code, you still need to overload all selectors one by one. With what I suggest, you declare variables (kinda like you did) in a `@media` block and that's it.

However, you'd still need to replace the original code with the variables. Let me show an example.

```css
@media (prefers-color-scheme: dark) {
    :root {
        --body-background-color: #181a1b;
        --color: #d5cbc6;
    }
}
```

Then in the original `body`, you replace its content with:

```css
body {
    background-color: var(--body-background-color);
    color: var(--color);
}
```